src/components/Tables/TableFilterBar.tsx

```diff
@@ -1,33 +1,74 @@
 "use client";
 
 import React from "react";
+import { styled } from "styled-components";
+import FilterAccordion from "@/components/Tables/Accordion";
 
-export interface FilterText {
+export type FilterText = {
```

Comment:
why did you change this? what's the advantage of interface over type?
(I think I prefer interface over type unless needed, but if there's other benefits I can be convinced!)

Reply:
Types are (imo) strictly better. You can compose them with & and |; they can alias primitives, arrays, tuples, functions; and many of typescripts utility types (https://www.typescriptlang.org/docs/handbook/utility-types.html) only work on types. In this case both would work, but interfaces have a strict subset of functionality and (again imo) should never really be used.

(also not sure why but `type` has superior intellisense in vscode, not sure how intended that is)

Reply:
![image](https://github.com/VauxhallFoodbank/vauxhall-foodbank/assets/104017547/fa647e6c-9c9f-45a7-8de9-e113a783dfd7)
disagree slightly, I think `type` can alias strings etc better but should only 'really' be used as an alias for existing types, whereas interfaces describe 'classes'/objects better (since often they are used in OOP) and the 'strict subset' stuff is usually desired.

I would much rather you stick to consistent use of interfaces wherever possible, but if you can't do a problem using interfaces then revert to types, but ideally I'd like to only use interfaces!

Reply:
Switched